--- mapserver_3.5/configure.in.orig	Wed Dec 12 01:56:46 2001
+++ mapserver_3.5/configure.in	Thu May 30 01:11:20 2002
@@ -1292,14 +1292,6 @@
     dnl This is a temporary error messgae until we figure the problem with
     dnl PHP4's regex
     dnl
-    if test "$PHP_VERSION_FLAG" = "-DPHP4" ; then
-      AC_MSG_ERROR([
-!!! The current version of PHP MapScript has some problems with       !!!
-!!! PHP4's bundled regex.  Until we figure the solution to the        !!!
-!!! problem, the workaround is to compile PHP4 with the system regex  !!!
-!!! Please re-configure and re-compile PHP4 with --with-system-regex  !!!
-!!! and then re-configure and re-compile MapServer.                   !!!])
-    fi
 
     PHP_REGEX_DIR="$PHP_SRC_DIR/regex"
     PHP_REGEX_INC="-I$PHP_REGEX_DIR"
--- mapserver_3.5/aclocal.m4.orig	Tue Jul 17 23:30:56 2001
+++ mapserver_3.5/aclocal.m4	Sat Jun  1 13:39:21 2002
@@ -274,13 +274,13 @@
 fi
 
 # if the greps and cuts don't do the job, set these manually
-PERL_CC="`grep ' cc=' perl.out | cut -d, -f1 | cut -d\' -f2`"
-PERL_OPT="`grep ' optimize=' perl.out | cut -d, -f2 | cut -d\' -f2`"
-PERL_CCFLAGS="`grep ' ccflags' perl.out | cut -d, -f1 | cut -d\' -f2`"
-PERL_LD="`grep ' ld=' perl.out | cut -d, -f1 | cut -d\' -f2`"
-PERL_LFLAGS="`grep ' cccdlflags=' perl.out | cut -d, -f1 | cut -d\' -f2`"
-PERL_CCDLFLAGS="`grep ' ccdlflags=' perl.out | cut -d, -f4 | cut -d\' -f2 | sed 's, ,,'`"
-PERL_LDDLFLAGS=`grep ' lddlflags' perl.out | cut -d, -f2 | cut -d\' -f2`
+PERL_CC="`perl -V:cc | cut -d\' -f2`"
+PERL_OPT="`perl -V:optimize | cut -d\' -f2`"
+PERL_CCFLAGS="`perl -V:ccflags | cut -d\' -f2`"
+PERL_LD="`perl -V:ld | cut -d\' -f2`"
+PERL_LFLAGS="`perl -V:cccdlflags | cut -d\' -f2`"
+PERL_CCDLFLAGS="`perl -V:ccdlflags | cut -d\' -f2`"
+PERL_LDDLFLAGS=`perl -V:lddlflags | cut -d\' -f2`
 #--------
 
 #if test -n "$PERL_CCDLFLAGS" ; then
--- mapserver_3.5/mapscript/php3/php_mapscript.c.orig	Tue Nov 27 16:39:33 2001
+++ mapserver_3.5/mapscript/php3/php_mapscript.c	Sat Jun  1 13:58:45 2002
@@ -431,6 +431,7 @@
 
 
 php3_module_entry php3_ms_module_entry = {
+    STANDARD_MODULE_HEADER,
     "MapScript", php3_ms_functions, php3_init_mapscript, php3_end_mapscript,
     NULL, NULL,
 #ifdef ZEND_VERSION
@@ -438,6 +439,7 @@
 #else
     php3_info_mapscript,
 #endif
+    NO_VERSION_YET,
     STANDARD_MODULE_PROPERTIES 
 };
 
