--- mapserver-3.6.4/configure.in.orig	Fri Feb 28 23:37:49 2003
+++ mapserver-3.6.4/configure.in	Fri Feb 28 23:38:46 2003
@@ -1374,14 +1374,6 @@
     dnl This is a temporary error messgae until we figure the problem with
     dnl PHP4's regex
     dnl
-    if test "$PHP_VERSION_FLAG" = "-DPHP4" ; then
-      AC_MSG_ERROR([
-!!! The current version of PHP MapScript has some problems with       !!!
-!!! PHP4's bundled regex.  Until we figure the solution to the        !!!
-!!! problem, the workaround is to compile PHP4 with the system regex  !!!
-!!! Please re-configure and re-compile PHP4 with --with-system-regex  !!!
-!!! and then re-configure and re-compile MapServer.                   !!!])
-    fi
 
     PHP_REGEX_DIR="$PHP_SRC_DIR/regex"
     PHP_REGEX_INC="-I$PHP_REGEX_DIR"
--- mapserver-3.6.4/aclocal.m4.orig	Fri Feb 28 23:38:56 2003
+++ mapserver-3.6.4/aclocal.m4	Fri Feb 28 23:39:30 2003
@@ -274,13 +274,13 @@
 fi
 
 # if the greps and cuts don't do the job, set these manually
-PERL_CC="`grep ' cc=' perl.out | cut -d, -f1 | cut -d\' -f2 | grep -v undef`"
-PERL_OPT="`grep ' optimize=' perl.out | cut -d, -f2 | cut -d\' -f2 | grep -v undef`"
-PERL_CCFLAGS="`grep ' ccflags' perl.out | cut -d, -f1 | grep cflags | cut -d\' -f2 | grep -v undef` `grep ' ccflags' perl.out | cut -d, -f2 | grep ccflags | cut -d\' -f2 | grep -v undef`"
-PERL_LD="`grep ' ld=' perl.out | cut -d, -f1 | cut -d\' -f2 | grep -v undef`"
-PERL_LFLAGS="`grep ' cccdlflags=' perl.out | cut -d, -f1 | cut -d\' -f2 | grep -v undef`"
-PERL_CCDLFLAGS="`grep ' ccdlflags=' perl.out | cut -d, -f4 | cut -d\' -f2 | sed 's, ,,' | grep -v undef`"
-PERL_LDDLFLAGS=`grep ' lddlflags' perl.out | cut -d, -f2 | cut -d\' -f2 | grep -v undef`
+PERL_CC="`perl -V:cc | cut -d\' -f2`"
+PERL_OPT="`perl -V:optimize | cut -d\' -f2`"
+PERL_CCFLAGS="`perl -V:ccflags | cut -d\' -f2`"
+PERL_LD="`perl -V:ld | cut -d\' -f2`"
+PERL_LFLAGS="`perl -V:cccdlflags | cut -d\' -f2`"
+PERL_CCDLFLAGS="`perl -V:ccdlflags | cut -d\' -f2`"
+PERL_LDDLFLAGS=`perl -V:lddlflags | cut -d\' -f2`
 #--------
 
 #if test -n "$PERL_CCDLFLAGS" ; then
